### ISSUE TITLE
New note/to-do disappear gradually

### ISSUE THREAD

**Issue by sealch** [laurent22/joplin#2829]
When clicking fast on New note/New to-do button new items are adding to the list and then disappearing  gradually

![Screen Recording 2020-03-21 at 18 14 06](https://user-images.githubusercontent.com/28362310/77231004-d9b9e700-6ba0-11ea-92fd-56c5e0c38268.gif)

## Environment

Joplin version: v1.0.194
Platform: macOS
OS specifics: 10.15

## Steps to reproduce

1. Open Joplin app
2. Start clicking fast on New note/New to-do button
3. New notes/to-do's are created and then disappear

## Describe what you expected to happen
I expected to create multiple empty notes/to-do's or warning that says that I can't create empty note/to-do if I already have one

## Logfile
```
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Scheduling sync operation..."
2020-03-21 18:29:18: "Provisional was not modified - deleting it"
2020-03-21 18:29:19: "Scheduling sync operation..."
2020-03-21 18:29:19: "Provisional was not modified - deleting it"
2020-03-21 18:29:19: "Scheduling sync operation..."
2020-03-21 18:29:19: "Scheduling sync operation..."
2020-03-21 18:29:19: "Provisional was not modified - deleting it"
2020-03-21 18:29:19: "Scheduling sync operation..."
2020-03-21 18:29:19: "Scheduling sync operation..."

**Comment by amitsin6h** [comment]
I will fix this

**Comment by amitsin6h** [comment]
So according to me, there is 2 way to fix this issue:
1.  First, count empty note inside the notebook and if empty not is present then do not allow the user to create another empty note

2. Second, avoid rapid clicking on New note button so that no empty new note will be created 

@tessus your thoughts on this?

**Comment by tessus** [comment]
This should have been fixed by 4920f648164140f22aab82009c7edbf52093197e

I can only reproduce this, if I jack-hammer my mouse button, so I'm not sure how often such a problem ever occurs. IMO, not a very good use of our time.

**Comment by amitsin6h** [comment]
I think then this issue should be closed

**Comment by tessus** [comment]
I leave this for Laurent to decide.

**Comment by sealch** [comment]
> This should have been fixed by [4920f64](https://github.com/laurent22/joplin/commit/4920f648164140f22aab82009c7edbf52093197e)
> 
> I can only reproduce this, if I jack-hammer my mouse button, so I'm not sure how often such a problem ever occurs. IMO, not a very good use of our time.

I think the main problem is that note/to-do creates and then disappears without any warning or message. So as amitsin6h said, there should be warning or new note/to-do should be creating. And you don't have to spam really fast to accomplish this behavior.

**Comment by laurent22** [comment]
No need to be fixed indeed. New notes disappear if they aren't modified.
